### ISSUE TITLE
Non training Batch Norm operator has bad performance for it running into tensorflow's non fused batch norm API

### ISSUE THREAD

**Issue by zhimingwang36** [keras-team/keras#10058]
In current Keras code with Tensorflow as it’s backend, the non-training batch norm operator/layer will finally run into Tensorflow’s old non-fused batch norm API (tf.nn.batch_normalization), which result in bad performance in both CPU and GPU.  
Tensorflow’s tf.nn.batch_normalization is non-fused batch norm, does computations using several individual Ops, While Tensorflow’s tf.nn.fused_batch_norm is fused batch norm which is just a new implementation that comprise several ops into one, has much better performance.

Following code shows the place where calling to Tensorflow's non fused batch norm happen, and suggest to call Tensorflow's fused Batch Norm API.
`def batch_normalization(x, mean, var, beta, gamma, epsilon=1e-3):
    """Applies batch normalization on x given mean, var, beta and gamma.

    I.e. returns:
    `output = (x - mean) / (sqrt(var) + epsilon) * gamma + beta`

    # Arguments
        x: Input tensor or variable.
        mean: Mean of batch.
        var: Variance of batch.
        beta: Tensor with which to center the input.
        gamma: Tensor by which to scale the input.
        epsilon: Fuzz factor.

    # Returns
        A tensor.
    """
    return tf.nn.batch_normalization(x, mean, var, beta, gamma, epsilon) ###Change to call tf.nn. batch_normalization() for better performance` 

Thanks.
